src/Library/demos/Grid/main.json

```
@@ -0,0 +1,12 @@
+{
+  "name": "Grid",
+  "category": "user_interface",
+  "description": "Allow people to perform using a grids",
+  "panels": [
+    "code",
+    "style",
+    "ui",
+    "preview"
```

Should be enough to see quickly what Grid is about:

```suggestion
    "ui",
    "preview"
```